Sources/Fuzzilli/Util/VariableMap.swift

```diff
@@ -55,6 +55,10 @@ public struct VariableMap<Value>: Sequence {
     public func contains(_ variable: Variable) -> Bool {
         return elements.count > variable.number && elements[variable.number] != nil
     }
+
+    public func hasHoles() -> Bool {
+        return elements.contains(where: {$0 == nil})
```

Comment:
Could this also just be `elements.contains(nil)` instead?

Reply:
I tried using contains(nil) but that throws a compile error: referencing instance method 'contains' on 'Optional' requires that 'Value' conform to 'Equatable' which is why I resorted to using contains(where:)

Reply:
Ah weird, well in that case ignore my comment :)